ribs/visualize.py

```diff
@@ -160,7 +163,12 @@ def grid_archive_heatmap(archive,
                       **pcm_kwargs)
 
     # Create the colorbar.
-    ax.figure.colorbar(t, ax=ax, pad=0.1)
+    if cbar == True:
+        print("Plotting cbar on default figure axis")
+        ax.figure.colorbar(t, ax=ax, pad=0.1)
```

Comment:
Don't add `pad=0.1` -- instead, we should add `cbar_kwargs` just like we add `pcm_kwargs`. We don't want to force colorbar options onto people.